src/ApiFetch.js

```diff
@@ -0,0 +1,15 @@
+import { getAllMovies } from './actions';
+
+const getData = async () => {
+  const url = '';
+  try {
+    const response = await fetch(url);
+    const parsed = await response.json();
+    getAllMovies(parsed.results);
```

Comment:
Do we call getAllMovies here and then getData in App or do we call getAllMovies in App with getData as an argument? Redux is confusing.